src/Domain/CoordinateMaps/Interval.hpp

```diff
@@ -1,135 +1,130 @@
-// Distributed under the MIT License.
```

Comment:
This file got completely rewritten in this commit. Is this perhaps related to line endings?